pavelib/quality.py

```diff
@@ -680,7 +681,12 @@ def _get_count_from_last_line(filename, file_type):
     This will return the number in the last line of a file.
     It is returning only the value (as a floating number).
     """
-    last_line = _get_report_contents(filename, file_type, last_line_only=True).strip()
+    report_contents = _get_report_contents(filename, file_type, last_line_only=True)
+
+    if report_contents is None:
+        return 0
```

Comment:
Returning 0 as the count if there is no content. Curious if this generates concern from anyone.

Reply:
@jmbowman fyi.

Reply:
How does this fix the problem if the _get_report_contents line is throwing an error?  It'll still throw an error, right?  Seems like you'd need a try/catch...

Reply:
Indeed, this generates concern

Reply:
As it turns out, if the `stylelint` command is not found, then the report ends up being empty.